### DISCUSSION TITLE
is there a limit on how many objects you can add into an object groups

### DISCUSSION THREAD

**Discussion by utami1511** [PaloAltoNetworks/pan-os-python#316] [Q&A] [answered]
Hello,

is there a limit on how many objects you can add when creating an object groups?

I am trying to add the data below:
Group_Name:  ag_accertify_inet
Group_Member (members separated by space): 
Accertify_INET_199.189.160.101 Accertify_INET_199.189.160.103   Accertify_INET_199.189.160.110 Accertify_INET_199.189.160.150   Accertify_INET_199.189.160.31 Accertify_INET_199.189.160.60   Accertify_INET_199.189.160.91 Accertify_INET_199.189.161.105

The data is read from a s/sheet and my code is as below:
for item in data['spreadsheet_New Address Group Objects']:
    grp_name = item['Name']
    grp_member = item['Address']
    group_object = panos.objects.AddressGroup(name=grp_name,static_value=grp_member)
    firewall.add(group_object)
    group_object.create()

And the error that I got is:
panos.errors.PanDeviceXapiError:  ag_accertify_inet -> static Node can be at most 63 characters, but current length: 244 value: Accertify_INET_199.189.160.101 Accertify_INET_199.189.160.103 Accertify_INET_199...
 ag_accertify_inet -> static is invalid

**Comment by shinmog** [comment, marked as answer]
Yeah, static address groups can have at most 63 members it looks like.
